### PR TITLE
[sample_encode] Init mfxExtMVCSeqDesc for vpp

### PR DESCRIPTION
**Pull request by Alexandr-Konovalov**
Since a2f2115c mfxExtMVCSeqDesc in m_mfxVppParams is empty.

Issue: MDP-65303

### CODE REVIEW

File: samples/sample_encode/src/pipeline_encode.cpp [hunk 1]
```diff
@@ -343,15 +343,21 @@ mfxStatus sTask::Reset()
 }
 
 mfxStatus CEncodingPipeline::AllocateExtMVCBuffers()
```
Comment (dmitryermilov) [inline samples/sample_encode/src/pipeline_encode.cpp:345]: can we fully drop this function? It's used in one place.

Comment (dmitryermilov) [inline samples/sample_encode/src/pipeline_encode.cpp:345]: Can we fully delete this function? It's used in one place only.

**Review by dmitryermilov — CHANGES_REQUESTED** [review]

File: samples/sample_encode/src/pipeline_encode.cpp (hunk 1 above)
Comment (Alexandr-Konovalov) [reply samples/sample_encode/src/pipeline_encode.cpp:345]: [CRegionEncodingPipeline::Init()](https://github.com/Intel-Media-SDK/MediaSDK/blob/c22544bcf12de3f7c4ab00276d0fb604fea5892b/samples/sample_encode/src/pipeline_region_encode.cpp#L420) is another place. Not that one is very useful.

Comment (dmitryermilov) [reply samples/sample_encode/src/pipeline_encode.cpp:345]: Why another? As far as I see this is the only place where it's called. I propose to remove

Comment (Alexandr-Konovalov) [reply samples/sample_encode/src/pipeline_encode.cpp:345]: @dmitryermilov 1st place is in [CEncodingPipeline::Init()](https://github.com/Intel-Media-SDK/MediaSDK/blob/c22544bcf12de3f7c4ab00276d0fb604fea5892b/samples/sample_encode/src/pipeline_encode.cpp#L1829). Sure, let's remove.

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**